src/lib/widgets/layers-toggle/sub-layers-toggle.component.ts

```diff
@@ -0,0 +1,28 @@
+import { Component, OnInit, Input} from '@angular/core';
+
+@Component({
+  selector: 'esri-sublayersToggle',
+  template: `<!-- Only display if there is a layer with sublayers -->
+    <div *ngIf="layer && layer.sublayers && layer.sublayers.items">
```

Comment:
Instead of doing this if check here, please move the `*ngIf` out to the layers-toggle.component.ts file. Then, instead of passing in the layer to this component you can just pass in the array of sublayers. This will allow this component to assume that if it has all the data that it needs and it can simply render the array of sublayers.